tests/models/test_product_live_time.py

```diff
@@ -0,0 +1,71 @@
+"""Unit tests for ProductLiveTime."""
+
+import datetime
+
+from tests.testutils import db
+from timed_release.models import product_live_time
+
+
+def test_to_dict():
+    """Test that the to dict function only returns the expected properties."""
+    test_timed_release = product_live_time.ProductLiveTime(
+        product_id='12',
+        time_of_day_product='00:00:01',
+        time_zone='GMT',
+        store_id=1)
+    expected_dict = {
+        'product_id': test_timed_release.product_id,
+        'time_of_day_product': test_timed_release.time_of_day_product,
+        'time_zone': test_timed_release.time_zone,
+        'store_id': test_timed_release.store_id}
+
+    assert test_timed_release.to_dict() == expected_dict
+
+
+@db.test_schema
+def test_get_product_live_time_details_success():
+    """Test to get product live time by product id."""
+    product_id = '12'
+    data = {
+        'product_id': product_id,
+        'time_of_day_product': datetime.time(20, 30, 00),
+        'time_zone': 'GMT',
+        'store_id': 1}
+    expected_response = {
+        'product_id': int(product_id),
+        'time_of_day_product': '20:30:00',
+        'time_zone': 'GMT',
+        'store_id': 1}
+    product_live_time.add_product_live_time_details(data)
+    response = product_live_time.get_product_live_time_details(product_id)
+    assert response.message == expected_response
+
+
+@db.test_schema
+def test_get_product_live_time_details_success_not_found():
+    """Test that an existing product id s not found."""
+    product_id = '12'
+    response = product_live_time.get_product_live_time_details(product_id)
+
+    assert response.status == 404
+    assert response.errors['code'] == 'not_found_error'
+
+
+@db.test_schema
+def test_validation_for_greater_than_zero():
```

Comment:
do the validations using parametrize - ref -https://github.com/kancha-supriya/microservice/commit/e4d87e9cc99419e98b9c9803b74dd427af067ebd#diff-f57c817bcce6d19c3603f065fce04b21R33